src/styles/utils/variables.scss

```diff
@@ -0,0 +1,9 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-text-color: #fff;
+$button-hover-text-color: #00acdc;
+$button-background-color: #00acdc;
+$button-hover-color: #fff;
+$card-border-color: #f3f3f3;
+$button-hover-border-color: #00acdc;
```

Comment:
You repeat `#00acdc` and white color 3 times, it should be only one, for example, `main-color`/`secondary-color` or something similar. Usually, you have a pallet and use the corresponding colors for different elements